lib/Config.js

```diff
@@ -1284,11 +1287,27 @@ class Config extends EventEmitter {
     }
 }
 
+// get the version ID encoding type
+function getVersionIdEncType() {
+    let encType;
+    if (process.env.VERSION_ID_ENC_TYPE !== undefined) {
```

Comment:
are the possible values 0 or 1?
If yes, will note for documentation purposes.

Reply:
Yes, but I think we need set ENC_TYPE_HEX by default. I am just trying to pass the tests with BASE62 then I will set HEX by default (for now).